Admin/Behaviors/AdminTrait.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace LAG\AdminBundle\Admin\Behaviors;
+
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\QueryBuilder;
+use LAG\AdminBundle\Admin\Configuration\AdminConfiguration;
+use LAG\AdminBundle\Admin\ManagerInterface;
+use Pagerfanta\Pagerfanta;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+trait AdminTrait
+{
+    /**
+     * Admin name.
+     *
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * Full namespace for Admin entity.
+     *
+     * @var string
+     */
+    protected $entityNamespace;
+
+    /**
+     * Entity repository.
+     *
+     * @var EntityRepository
+     */
+    protected $repository;
+
+    /**
+     * Form type.
+     *
+     * @var string
+     */
+    protected $formType;
+
+    /**
+     * Controller.
+     *
+     * @var Controller
+     */
+    protected $controller;
+
+    /**
+     * @var Pagerfanta
+     */
+    protected $pager;
+
+    /**
+     * @var QueryBuilder
+     */
+    protected $queryBuilder;
+
+    /**
+     * Entity manager (doctrine entity manager by default).
+     *
+     * @var ManagerInterface
+     */
+    protected $manager;
+
+    /**
+     * @var AdminConfiguration
+     */
+    protected $configuration;
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getEntityNamespace()
+    {
+        return $this->entityNamespace;
+    }
+
+    /**
+     * @return EntityRepository
+     */
+    public function getRepository()
+    {
+        return $this->repository;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getFormType()
+    {
+        return $this->formType;
+    }
+
+    /**
+     * @return ManagerInterface
+     */
+    public function getManager()
+    {
+        return $this->manager;
+    }
+
+    /**
+     * @return AdminConfiguration
+     */
+    public function getConfiguration()
+    {
+        return $this->configuration;
+    }
+
+    /**
+     * @return Pagerfanta
+     */
+    public function getPager()
+    {
+        return $this->pager;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getController()
+    {
+        return $this->controller;
+    }
+
+    public function getEntityLabel()
```

Comment:
Also, does the admin should know which method is used to show the label? Is this configurable?
If not already done, an Interface should ensure that method mlust be implemented.